### DISCUSSION TITLE
System > Cameras output meaning

### DISCUSSION THREAD

**Discussion by bucklevision** [blakeblackshear/frigate#8820] [General]
I have a few questions around the output on the system page. My cameras provide 2 streams, 1 is a low res 4fps and the second is a 1080p 18fps feed. I use the low res feed for detect and high for record roles. 

In the system page it shows the ffmpeg, capture and detect fps values. My ffmpeg and capture fps is always the speed of the detect stream (4fps) but the detect always shows 0 (0 skipped). I don't know whether this is correct or not. Is there any description of the fields and their ideal values?

**Comment by NickM-27** [comment]
The fps are correct, detect fps is only going to be > 0 when object detection is being run
